internal/services/postgres/postgresql_flexible_server_resource_test.go

```diff
@@ -599,7 +599,7 @@ resource "azurerm_postgresql_flexible_server" "test" {
   administrator_password = "123wsxQAZ"
   version                = "13"
   backup_retention_days  = 10
-  storage_mb             = 65536
+  storage_mb             = 4193280
```

Comment:
this'd cause a huge cost increase for this test - can we revert this?

Reply:
Updated